### PR TITLE
Inconsistencies with Cluster metrics across the UI

### PR DESCRIPTION
**Pull request by aalves08**
Addresses Github issue: [#5430](https://github.com/rancher/dashboard/issues/5430)
Addresses Zube issue: [#5456](https://zube.io/rancher/dashboard-ui/c/5456)

- Fix number of PODS on homepage cluster table by displaying the proper PODs usage (`PodsUsage` formatter)
- Fix units for MEM on homepage cluster table to always be GiB (consistent with the rest of the UI)
- Show **total** number of nodes rather that **useful** nodes in cluster dashboard view 
- Remove reserved info for CPU and MEM in cluster table in homepage 
- Fix PODs usage in nodes list view (matches cluster dashboard and cluster table in homepage)
- Display N/A when usage is zero for CPU, MEM, PODS in cluster `nodes list view`

**Preview**

_homepage_
Screenshot 2022-03-29 at 14 25 25

_nodes list view_
Screenshot 2022-03-29 at 14 25 14

_cluster dashboard_
Screenshot 2022-03-29 at 14 25 41

### CODE REVIEW

File: components/formatter/PercentageBar.vue [hunk 1]
```diff
@@ -30,5 +30,12 @@ export default {
 </script>
 
 <template>
-  <PercentageBarComponent :value="percentage" :show-percentage="true" />
+  <p v-if="!value || value === '0'">
+    N/A
```
Comment (nwmac) [inline components/formatter/PercentageBar.vue:34]: Should probably use a localised string for this

**Review by nwmac — CHANGES_REQUESTED** [review]
One minor comment.

We should also make the pod usage formatter a delayed component once the table performance changes are in

**Comment by aalves08** [comment]
> One minor comment.
> 
> We should also make the pod usage formatter a delayed component once the table performance changes are in

Once you merge your end @nwmac I will test everything on my side. The idea then is to add `delayLoading: true` to the headers col definition and also add the method `startDelayedLoading()` to the formatter.

One question though:
I need to "move" what's inside of the async fetch in `components/formatter/PodsUsage.vue` to be called from `startDelayedLoading()` method? Those are the only two changes to do once your part gets merged, right?

Let's coordinate so that we can use this for 2.6.5 🙏

**Review by nwmac — APPROVED** [review]
LGTM - You'll need this PR merged - https://github.com/rancher/dashboard/pull/5635 - to fix an issue with the sortable table delay loading

**Pull request merged**